projects/noloan2/app/src/main/java/hasadna/noloan2/RecyclerAdapter.java

```diff
@@ -0,0 +1,63 @@
+package hasadna.noloan2;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import hasadna.noloan2.protobuf.SMSProto.SmsMessage;
+import noloan.R;
+
+public class RecyclerAdapter extends RecyclerView.Adapter<RecyclerAdapter.RecyclerViewHolder> {
```

Comment:
This name is too generic. Sounds like an Android library class. How about we rename to `SmsRecyclerAdapter`?